tools/afl-fuzzer/qemu-bridge.c

```diff
@@ -0,0 +1,395 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+// Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+
+/*
+ * Creates an IO bridge between two QEMU instances where messages can be passed
+ * between the parent and child instances via messages queues and shared memory.
+ *
+ * The parent is usually the QEMU instance that runs the operating system (like
+ * Linux) on the application processor whilst the child is typically a smaller
+ * processor running an embedded firmware. The parent and child do not need to
+ * be the same architecture but are expected to communicate over a local bus.
+ */
+
+#include <mqueue.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <fcntl.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <glib.h>
+#include <errno.h>
+#include "qemu-bridge.h"
+
+/* set to 1 to enable debug */
+static int io_bridge_debug;//QEMU_IO_DEBUG;
```

Comment:
cmd line option.

Reply:
I didn't understand what you were trying to say. Can you give some more context. Thanks!

Reply:
I think Liam wants this field to be able to be set through a command line argument rather than programatically